addons/udes_stock/models/stock_picking_type.py

```diff
@@ -1,10 +1,41 @@
-"""UDES core picking type functionality."""
 from odoo import fields, models
 
 
+TARGET_STORAGE_FORMAT_OPTIONS = [
+    ("pallet_products", "Pallet of products"),
+    ("pallet_packages", "Pallet of packages"),
+    ("package", "Packages"),
+    ("product", "Products"),
+]
+
+
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    u_user_scans = fields.Selection(
+        [("pallet", "Pallets"), ("package", "Packages"), ("product", "Products")],
+        string="What the User Scans",
+        help="What the user scans when asked to scan something from pickings of this type",
+    )
+    u_target_storage_format = fields.Selection(
+        TARGET_STORAGE_FORMAT_OPTIONS, string="Target Storage Format"
+    )
+    u_under_receive = fields.Boolean(
+        string="Under Receive",
+        default=False,
+        help="If True, allow less items than the expected quantity in a move line.",
+    )
+    u_over_receive = fields.Boolean(
+        string="Over Receive",
+        default=True,
+        help="If True, allow additional items not in the ASN, or over the expected quantity, "
```

Comment:
Suggestion: The help string from v11 is referencing ASN and Goods In, but this setting applies for other types too.

I suggest just:

`If True, allow more items than the expected quantity in a move line.`